Configuration/StandardSequences/python/GeometryConf.py

```diff
@@ -32,6 +32,7 @@
     'TrackerOnly' : 'Configuration.Geometry.GeometrySimTracker_cff,Configuration.Geometry.GeometryRecoTracker_cff',
     'HCal'        : 'Configuration.Geometry.GeometrySimHCAL_cff,Configuration.Geometry.GeometryRecoHCAL_cff',
     'Extended2023sim'   : 'Extended2023sim,Extended2023simReco',
-    'Extended2023dev'   : 'Extended2023dev,Extended2023devReco'
+    'Extended2023dev'   : 'Extended2023dev,Extended2023devReco',
+    'Extended2023LReco'   : 'Extended2023LReco,Extended2023LRecoReco'
```

Comment:
@ianna - the expectation is that they do not remain the same for long (in fact if they do, we just drop the 2023sim scenario)

> On Apr 5, 2016, at 12:19 PM, Ianna Osborne <notifications@github.com> wrote:
> 
> In Configuration/StandardSequences/python/GeometryConf.py:
> 
> > @@ -32,6 +32,7 @@
> >      'TrackerOnly' : 'Configuration.Geometry.GeometrySimTracker_cff,Configuration.Geometry.GeometryRecoTracker_cff',
> >      'HCal'        : 'Configuration.Geometry.GeometrySimHCAL_cff,Configuration.Geometry.GeometryRecoHCAL_cff',
> >      'Extended2023sim'   : 'Extended2023sim,Extended2023simReco',
> > -    'Extended2023dev'   : 'Extended2023dev,Extended2023devReco'
> > +    'Extended2023dev'   : 'Extended2023dev,Extended2023devReco',
> > +    'Extended2023LReco'   : 'Extended2023LReco,Extended2023LRecoReco'
> 
> @boudoul - perhaps, the following is better as at this point the configuration files are identical:
> 
> 'Extended2023LReco' : 'Extended2023sim,Extended2023simReco'
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>